plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergOrcMetricsCollection.java

```diff
@@ -129,7 +249,7 @@ public void testBasic()
         assertQuery("SELECT min(orderpriority) FROM tpch.tiny.orders", "VALUES '" + lowerBounds.get(6) + "'");
         assertQuery("SELECT min(clerk) FROM tpch.tiny.orders", "VALUES '" + lowerBounds.get(7) + "'");
         assertQuery("SELECT min(shippriority) FROM tpch.tiny.orders", "VALUES " + lowerBounds.get(8));
-        assertQuery("SELECT min(comment) FROM tpch.tiny.orders", "VALUES '" + lowerBounds.get(9) + "'");
+        assertQuery("SELECT substring(min(comment), 1, 16) FROM tpch.tiny.orders", "VALUES '" + lowerBounds.get(9) + "'");
```

Comment:
`lowerBounds.get(9)` is the actual, while `SELECT substring(min(comment), 1, 16) FROM tpch.tiny.orders` is the expected. The expected/actual are reversed here.

Reply:
@findepi I tried reversing the order for all the `assertQuery` in the test function but encountered failures when using `SELECT substring(min(comment), 1, 16) FROM tpch.tiny.orders` as the expected sql statement. Checked other test files and looks like they placed all `select` statement as the expected and result as the actual.

Reply:
we shouldn't use `assertQuery` at all, when we want to assert a value that we have at hand.

this is preexisting, so ok to followup